apps/gjafakort/api/src/services/versionConfiguration.ts

```diff
@@ -17,9 +16,24 @@ const v2: Version = {
   type: 'gjafakort-user-2',
 }
 
-export const getVersionConfiguration = (): Version => {
-  const useV2 =
-    new Date() >= new Date(versionCutoffTime) || environment.isFerdagjof2Enabled
+const IS_FERDAGJOF_2_ENABLED = 'isferdagjof2enabled'
+
+const sdkKey = environment.configCat.sdkKey
+if (!sdkKey) {
+  throw new Error(
+    'Trying to initialize configcat client without configCat.sdkKey environment variable',
+  )
+}
+
+const configCat = eval('require')('configcat-node').createClient(sdkKey, {
```

Comment:
This could be problematic down the line with JS bundlers but since in this case is pretty short-lived, should be ok

Reply:
@pshomov I agree